tests/performance/test_phase3_performance.py

```
@@ -0,0 +1,972 @@
+            pytest.skip("psutil not available for memory testing")
+
+        streaming_reader = StreamingFileReader(chunk_size=8192)
+        metrics = PerformanceMetrics()
+
+        # Baseline memory
+        gc.collect()
+        baseline_memory = process.memory_info().rss
+        metrics.record_memory()
+
+        # Process large file multiple times
+        for i in range(10):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)